com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -264,6 +270,14 @@ void CollectGarbageOnDemand()
             return true;
         }
 #endif
+#if UNITY_IOS || UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX
+        void InitializeOpenGLCore() {
```

Comment:
Both OpenGLCore and Opengles call this function, batter rename it to `InitializeOpenGL()`

Reply:
yes, let me fix this :-)